README.md

```diff
@@ -258,15 +269,15 @@ The Joplin applications automatically save previous versions of your notes at re
 
 This feature can be disabled from the "Note history" section in the settings, and it is also possible to change for how long the history of a note is saved.
 
-More information about this feature in the [announcement post](https://www.patreon.com/posts/note-history-now-27083082).
+More information please see the [Note History page](https://github.com/laurent22/joplin/blob/dev/readme/note_history.md).
```

Comment:
The html documentation is generated from the .md files, thus I believe they are correctly translated during that process. I believe this is correct, but let me confirm with @laurent22: what's the correct reference?

Reply:
I believe @Daeraxa added the note_history file, so it's good to link to it directly rather than the blog post.